x/axelarnet/keeper/msg_server.go

```diff
@@ -340,6 +340,9 @@ func IBCTransfer(ctx sdk.Context, k types.BaseKeeper, t types.IBCTransferKeeper,
 	if err == nil {
 		// SendTransfer would return error if the next sequence not found
 		seq, _ := c.GetNextSequenceSend(ctx, portID, channelID)
+		if seq == 0 {
```

Comment:
I suggest handling the error returned by `GetNextSequenceSend` as well just for the sake of best practice if we are not handling it right now simply because some other methods are already calling it. Dependency upgrades may change things unnoticeably for us and cause issues down the way.

Reply:
@haiyizxx @jcs47

Reply:
makes sense